File: Packages/AnySSHKit/Sources/Fixtures/Data/blobs/sample.ts
// A recorded sample exercising every scope the golden table asserts.
export interface Waypoint {
    name: string;
    attempts: number;
}

export function retry(point: Waypoint, delay: number): boolean {
    const label = "retrying";
    return point.attempts < 3 && delay > 0.5 && label.length > 0;
}

export const origin: Waypoint = { name: "origin", attempts: 0 };

File: Packages/AnySSHKit/Sources/Fixtures/Data/blobs/sample.swift
import Foundation

/// A recorded sample exercising every scope the golden table asserts.
struct Waypoint {
    let name: String
    let attempts: Int

    func retry(after delay: Double) -> Bool {
        // A line comment the golden table reads.
        return attempts < 3 && delay > 0.5
    }
}

let origin = Waypoint(name: "origin", attempts: 0)
